pom.xml

```diff
@@ -48,13 +48,11 @@
       <groupId>net.sf.kxml</groupId>
       <artifactId>kxml2</artifactId>
       <version>2.3.0</version>
-      <scope>compile</scope>
```

Comment:
This was probably confusing things: `compile` is the default and doesn't need to be specified.